config/extension-crds/machineconfig.crd.yaml

```diff
@@ -0,0 +1,341 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I'm surprised we have to keep a copy of the the CRD. Is this done automatically by operator-sdk? What happens if MCO changes something on one of the CRDs? Do we have to check the version of the CRD or something like that?

Reply:
Using the OpenShift CRDs with `envtest` looks a bit convoluted. 
It took me fair amount of time to figure out a working scenario. I'm sure there will be a better way.

Please have a look at the following issues - 
https://github.com/kubernetes-sigs/controller-runtime/issues/1191
https://github.com/operator-framework/operator-sdk/issues/4434
https://github.com/kubernetes-sigs/controller-runtime/pull/1393

We do run the risk of test cases breaking or giving false positives if MCO changes the CRD and we don't update the same for the operator tests.

Reply:
I see. It's a known problem and this is the best we can do right now. Thanks!